BUILD

```diff
@@ -122,6 +122,19 @@ USE_UTMPX=1|0
     like "who" to work correctly (the service configuration items "inittab-id" and "inittab-line"
     have no effect if this is disabled). If not set to any value, support is enabled for certain
     systems automatically and disabled for all others.
+USE_INITGROUPS=1|0
```

Comment:
This change shouldn't be part of this merge req...

Reply:
Seems like you squashed other commit(s) into this one, you'll need to undo that

Reply:
The commit still contains the change.

Reply:
Also, don't merge master into your PR branches. Rebase your PR branch onto master instead. That will probably fix this anyway.